tsdat/io/writers.py

```diff
@@ -80,6 +80,10 @@ def write(
                     }
                 )
 
+        if "time" in dataset.dims:
+            if "unlimited_dims" not in dataset.encoding or "time" not in dataset.encoding["unlimited_dims"]:
+                to_netcdf_kwargs["unlimited_dims"] = ["time"]
```

Comment:
This will override the dataset's encoding if it is something like `["height"]`, which is probably fine in any case we'd see, but I'd prefer if we made that be `["time", "height"]` instead.

We can change the right side of line 85 to be `["time"] + dataset.encoding.get("unlimited_dims", [])` to correct this, I think.